tests/corpus/temp.cc

```diff
@@ -0,0 +1 @@
+// Hello World
```

Comment:
What is the purpose of this .cc file? Can we remove it?

I assume the corpus directory is for storing the fuzz corpus? Can we rename it `tests/fuzz_corpus`?

Reply:
Yes, you can remove it and rename it to `tests/fuzz_corpus`.